spec.html

```diff
@@ -23708,7 +23708,8 @@ <h1>parseFloat ( _string_ )</h1>
       <p>The `parseFloat` function produces a Number value dictated by interpretation of the contents of the _string_ argument as a decimal literal.</p>
       <p>The `parseFloat` function is the <dfn>%parseFloat%</dfn> intrinsic object. When the `parseFloat` function is called with one argument _string_, the following steps are taken:</p>
       <emu-alg>
-        1. Let _trimmedString_ be ? TrimString(_string_, `"start"`).
+        1. Let _inputString_ be ? ToString(_string_).
+        1. Let _trimmedString_ be ? TrimString(_inputString_, `"start"`).
```

Comment:
Nit: 

```suggestion
        1. Let _trimmedString_ be ! TrimString(_inputString_, `"start"`).
```

Sorry, I forgot to check this one before.